internal/controllers/staticroute_controller.go

```diff
@@ -59,6 +60,7 @@ type StaticRouteReconciler struct {
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.10.0/pkg/reconcile
 func (r *StaticRouteReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	l := log.FromContext(ctx).WithName("static-route-controller")
+	analytics.SendAnonymousInfo(fmt.Sprintf("reconciling Static route  %s", req.NamespacedName))
```

Comment:
Im not sure if recording the namespace the reconciliation took place in is good in terms of privacy

Reply:
we can ditch the name entirely